reference/sectionB_elements.inc

```diff
@@ -377,44 +377,44 @@ A :code:`math` element information item (referred to in this specification as a
+| Relational and Logical Operators | :code:`eq`, :code:`neq`, :code:`gt`, :code:`lt`, :code:`geq`,  |
+|                                  | :code:`leq`, :code:`and`, :code:`or`, :code:`xor`, :code:`not` |
++----------------------------------+----------------------------------------------------------------+
+| Arithmetic Operators             | :code:`plus`, :code:`minus`, :code:`times`,                    |
+|                                  | :code:`divide`, :code:`power`, :code:`root`,                   |
+|                                  | :code:`abs`, :code:`exp`, :code:`ln`, :code:`log`,             |
+|                                  | :code:`floor`, :code:`ceiling`, :code:`min`,                   |
+|                                  | :code:`max`, :code:`rem`,                                      |
++----------------------------------+----------------------------------------------------------------+
+| Calculus Elements                | :code:`diff`                                                   |
++----------------------------------+----------------------------------------------------------------+
+| Qualifier Elements               | :code:`bvar`, :code:`logbase`, :code:`degree` \*               |
++----------------------------------+----------------------------------------------------------------+
+| Trigonometric Operators          | :code:`sin`, :code:`cos`, :code:`tan`, :code:`sec`,            |
+|                                  | :code:`csc`, :code:`cot`,                                      |
+|                                  |                                                                |
+|                                  | :code:`sinh`, :code:`cosh`, :code:`tanh`, :code:`sech`,        |
+|                                  | :code:`csch`, :code:`coth`, :code:`arcsin`,                    |
+|                                  | :code:`arccos`, :code:`arctan`,                                |
+|                                  |                                                                |
+|                                  | :code:`arcsec`, :code:`arccsc`, :code:`arccot`,                |
+|                                  | :code:`arcsinh`, :code:`arccosh`, :code:`arctanh`,             |
+|                                  | :code:`arcsech`, :code:`arccsch`, :code:`arccoth`              |
++----------------------------------+----------------------------------------------------------------+
+| Mathematical and Logical         | :code:`pi`, :code:`exponentiale`,                              |
+| Constants                        | :code:`notanumber`, :code:`infinity`,                          |
+|                                  | :code:`true`, :code:`false`                                    |
++----------------------------------+----------------------------------------------------------------+
+
+\* (child of :code:`root` or :code:`diff`)
```

Comment:
Re the `*`, can you not simply remove it and, instead, have:

```
**Note:** :code:`degree` cannot be used on its own, but as a child of :code:`root` or :code:`diff`.
```

Reply:
It's not the star, it's the newlines ... watch this space ...